### ISSUE TITLE
Decomission the upnp container and bring it into hm-diag

### ISSUE THREAD

**Issue by vpetersson** [NebraLtd/hm-diag#287]
## Background

We currently have a [container](https://github.com/NebraLtd/hm-upnp) running on the miners with the sole purpose of enabling uPNP. We can quickly just sunset this in favor of a single command in this repo.

It is however worth noting that this will be unnecessary once we move to light miners, but the timeline for this keeps slipping.

## Acceptance Criteria

* Remove the hm-upnp container from `docker-compose.yml`
* Add the upnp package to the [Dockerfile](https://github.com/NebraLtd/hm-diag/blob/master/Dockerfile)
* Create a new [task](https://github.com/NebraLtd/hm-diag/blob/master/hw_diag/tasks.py) that runs every 12 hours that invokes the [command](https://github.com/NebraLtd/hm-upnp/blob/main/configureUpnp.py#L34)

**Comment by shawaj** [comment]
@vpetersson IMO this should be in hm-config not hm-diag.

Because UPnP requires host networking and so does config. Whereas diag doesn't

**Comment by shawaj** [comment]
Also @vpetersson I don't think we need to run UPnP every 12 hours. We can just run it once at container start up

**Comment by vpetersson** [comment]
Yeah, you're right. I was thinking this would solve the IP announcement with Helium, but those are two separate issues altogether. Just mixed them up in my head.

**Comment by MuratUrsavas** [comment]
So, are we moving this task into hm-config?

**Comment by MuratUrsavas** [comment]
What about this:

```python
while(regionID is None):
    # While no region set
    try:
        with open("/var/pktfwd/region", 'r') as regionOut:
            regionFile = regionOut.read()

            if(len(regionFile) > 3):
                print("Frequency: " + str(regionFile))
                regionID = str(regionFile).rstrip('\n')
                break
        print("Invalid Contents")
        sleep(60)
        print("Try loop again")
    except FileNotFoundError:
        print("File Not Detected, Sleeping")
        sleep(60)
```
Should we really wait for it before enabling UPnP?

**Comment by shawaj** [comment]
> So, are we moving this task into hm-config?

Yes we should. We don't want host networking in diag container IMO
